### DISCUSSION TITLE
Build failure fails to update deployment

### DISCUSSION THREAD

**Discussion by 1800joe** [noop-inc/documentation#94] [Bugs] [answered]
See deployment appears empty
<img width="1168" alt="Screenshot 2024-01-29 at 11 31 19 AM" src="https://github.com/noop-inc/documentation/assets/3111670/9852c0df-3b38-4e23-be23-ebf4ac2aa72e">

Build shows failed, but deployment is still in progress
<img width="540" alt="Screenshot 2024-01-29 at 11 31 28 AM" src="https://github.com/noop-inc/documentation/assets/3111670/a12452ac-1002-4c50-9d9d-7b72bb919daf">

Build has no log output
<img width="1147" alt="Screenshot 2024-01-29 at 11 31 34 AM" src="https://github.com/noop-inc/documentation/assets/3111670/7fb4041b-76f0-4584-b392-889640ad5e4b">

**Comment by derekwolpert** [comment, marked as answer]
@1800joe This situation should be addressed with Noop Desktop v0.5.0 and later. Please let us know if you are still encountering this issue, and we'll investigate further.
